bin/wp-enforcer

```diff
@@ -107,17 +123,22 @@ if [[ -f "$PROJECT"/.git/hooks/pre-commit ]]; then
     echo "A pre-commit hook already exists in $PROJECT/.git/hooks/pre-commit."
     if confirm "Overwrite the existing file?"; then
       copy_hook "pre-commit"
+      update_root_path
     fi
   else
     echo "pre-commit hook already exists, skipping"
   fi
 
 else
   copy_hook "pre-commit"
+  update_root_path
 fi
 
-# Copy the PHP_CodeSniffer standards file
-if [[ ! -f "$PROJECT"/phpcs.xml && "$ruleset" = false ]]; then
+# Copy the PHP_CodeSniffer standards file if WordPress standard is not available
+#
+if [[ "global" -eq ${PHPCS_PATH} && 0 -lt $(phpcs -i | grep -woe ${PHPCS_STANDARD} | wc -w) ]]; then
+  echo "WordPress standard is installed as global"
```

Comment:
It would probably be worth using the `${PHPCS_STANDARD}` variable in this message. It may not be the WP standard that's being used, and following the principle of least surprise seems like a good idea.